torch/nn/quantized/modules/linear.py

```diff
@@ -120,6 +120,7 @@ def __init__(self, in_features, out_features, bias_=True):
             [out_features, in_features], scale=1, zero_point=0, dtype=torch.qint8)
 
         self.set_weight(qweight)
+        self.weight_scale = 1.0
```

Comment:
May be we can add a check here to make sure that the qscheme of weights is kPerTensorAffine.